client/src/app/tabs/PropertiesContainer.js

```diff
@@ -18,48 +18,90 @@ import dragger from '../../util/dom/dragger';
 
 import css from './PropertiesContainer.less';
 
+import HandleBar from '../../../resources/icons/HandleBar.svg';
+
 import { throttle } from '../../util';
 
+export const MIN_WIDTH = 250;
+
 export const DEFAULT_LAYOUT = {
   open: false,
-  width: 250
+  width: MIN_WIDTH
 };
 
-export const MIN_WIDTH = 150;
-export const MAX_WIDTH = 650;
-
-
 /**
  * Container for properties panel that can be resized and toggled.
  */
 class PropertiesContainerWrapped extends PureComponent {
   constructor(props) {
     super(props);
 
-    this.handleResize = throttle(this.handleResize);
```

Comment:
I wonder whether we should still throttle this window resize function.

Reply:
I did a renaming of this function. `handleResize` tuned into `handlePanelResize` (which still has the throttle). And added a new `handleResize` to listen to the window resizing events (`window.addEventListener('resize', this.handleResize);`)

Reply:
Should we maybe add this listener to `componentDidMount` and also clean it up on `comoponentWillUnmount`?

Just the way we do in [`AppParent`](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/AppParent.js#L369), to ensure things get added and cleaned up properly.

Reply:
Makes sense. I changed this to be added and cleaned up on `componentDidMount`/`comoponentWillUnmount`